hnn_core/cell_response.py

```diff
@@ -381,7 +384,34 @@ def plot_spikes_hist(self, trial_idx=None, ax=None, spike_types=None,
         return plot_spikes_hist(self, trial_idx=trial_idx, ax=ax,
                                 spike_types=spike_types, show=show)
 
-    def write(self, fname):
+    def _write_hdf5(self, fname):
+        """Write spiking activity to a hdf5 file.
+
+        Parameters
+        ----------
+        fname : str
+            String format (e.g., 'spk.hdf5') of the
+            path to the output spike file.
+
+        Outputs
+        -------
+        A hdf5 file containing the cell response object
+        """
+        fname = str(fname)
+        # Hardcoded fname for test purposes
```

Comment:
Is this old? The `fname` doesn't seem to be hardcoded anymore

Reply:
Yes. Sorry I forgot to remove those comments.